src/clients/apiClient.ts

```diff
@@ -116,11 +127,10 @@ export async function fetchSightingById(sightingId: number): Promise<WhaleSighti
   }
 }
 
-export async function createSighting(newSighting: NewSighting, encodedUsernamePassword: string): Promise<Response> {
-  const response = await fetch(`${backendUrl}/sightings/submit`, {
+export async function createSighting(newSighting: NewSighting): Promise<Response> {
+  const response = await fetch(`https://${backendUrl}/sightings/submit`, {
```

Comment:
I don't think you meant to change these!
```suggestion
export async function createSighting(newSighting: NewSighting, encodedUsernamePassword: string): Promise<Response> {
  const response = await fetch(`${backendUrl}/sightings/submit`, {
```